src/app/shared/components/template/components/button/button.component.ts

```diff
@@ -7,7 +7,7 @@ import { FlowTypes } from "packages/data-models";
 interface IButtonParams {
   /** TEMPLATE PARAMETER: "variant" */
   variant:
-    | ""
+    | "default"
     | "alternative"
```

Comment:
@jfmcquade Rather than an empty string, I changed the default button `variant` to `default` which makes those buttons identifiable as well (esp in styling). Any reservations?

Reply:
I think that makes sense. The styles/variants are a bit of a mess, but I can see how it makes sense to have a `default` variant to target, especially for theme overrides.